share/man/man4/wlan.4

```diff
@@ -25,12 +25,12 @@
 .\"
 .\" $FreeBSD$
 .\"
-.Dd January 26, 2021
+.Dd April 19, 2023
 .Dt WLAN 4
 .Os
 .Sh NAME
 .Nm wlan
-.Nd generic 802.11 link-layer support
+.Nd generic Wifi 802.11 link-layer support
```

Comment:
WiFi is yhe industry standard.

Reply:
Thank you for your helpful comments, @emaste and @bsdimp.  I have updated my pull request accordingly.